include/psen_scan/default_parameters.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019 Pilz GmbH & Co. KG
```

Comment:
This file should have Copyright 2020, since it was created in 2020.
Other files created in 2019 and modified in 2020 should be updated with 2019-2020.